packages/cli/src/deprecation/deprecation.service.ts

```diff
@@ -47,6 +47,13 @@ export class DeprecationService {
 			message: 'MySQL and MariaDB are deprecated. Please migrate to PostgreSQL.',
 			checkValue: (value: string) => ['mysqldb', 'mariadb'].includes(value),
 		},
+		{
+			envVar: 'DB_TYPE',
```

Comment:
This deprecation will warn about `DB_TYPE` but that's not the env var that the user needs to change.